### PR TITLE
add more features / commands

### PR DESCRIPTION
**Pull request by ckxz105**
update readme for run-pipeline / add-text-embedding-feature / tee commands
add filter support for get-phrase-matches

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -683,3 +690,133 @@ in the Ground Truth File and the candidate is different from the corresponding k
...
+**Options:**
+- `-q / --sparql-query-endpoint`: The sparql query endpoint the sysetm should query to. Default is  offical wikidata query endpoint https://query.wikidata.org/sparql. Note: The official wikidata query endpoint has frequency and timeout limit.
+- `-o / --output-column-name`: The output scoring column name. If not provided, the name of the embedding model will be used.
+- `-g / --generate-projector-file`: If given, the function will generate the files needed to run the Google Project visualization.
+###### Following options are wrapped from kgtk, please refer to [here](https://github.com/usc-isi-i2/kgtk/blob/feature/embedding/kgtk/cli/text_embedding_README.md "here") for details.
+- `-c / --column-vector-strategy`: The centroid choosing method.
+- `-m / --embedding-model`: The pre-fitted models used for generating the vectors.
+- `-d / --distance-function`: The distance measurement function to used for scoring.
+- `-n`: The number of cells used to estimate the vector for a column.
+- `--label-properties`: The names of the eges for `label` properties
```
Comment (saggu) [inline README.md:736]: what is eges?

File: README.md [hunk 2]
```diff
@@ -683,3 +690,133 @@ in the Ground Truth File and the candidate is different from the corresponding k
...
+**File Example:**
+```
+tag         file      precision recall      f1
+gt-embed  v15_685.csv 0.473684211 0.473684211 0.473684211
+gt-embed  v15_686.csv 0.115384615 0.115384615 0.115384615
+```
+
+#### Implementation
+This command used python's subprocess to call shell functions then excute the corresponding shell codes.
+
```
Comment (saggu) [inline README.md:822]: add the new commands to the table of contents at the top of the README as well

File: tl/cli/get-phrase-matches.py [hunk 3]
```diff
@@ -29,16 +29,34 @@ def add_arguments(parser):
...
     parser.add_argument('input_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)
 
+    # used for filtering
+    parser.add_argument('--filter', action='store', nargs='?', dest='filter_condition',
+                        default="", help="If set to run filtering, which kind of the data should keep.")
+
 
 def run(**kwargs):
     from tl.candidate_generation import phrase_query_candidates
+    from tl.utility.filter import Filter
```
Comment (saggu) [inline tl/cli/get-phrase-matches.py:39]: this import should be in the phrase_query_candidates.py in the candidate_generation folder

Comment (ckxz105) [reply tl/cli/get-phrase-matches.py]: All 3 suggestions fixed

**Pull request merged**